nemo/deploy/query.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import typing
+
+import numpy as np
```

Comment:
Can we remove this file? We have it here https://github.com/NVIDIA/NeMo/blob/main/nemo/deploy/nlp/query_llm.py . Class name is different though.

Reply:
Removed and used NemoQueryLLM class from query_llm.py